concepts/05 Widgets/CheckBox/00 Overview.md

```diff
@@ -42,6 +42,50 @@ The following code adds the **CheckBox** to your page.
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxCheckBox
+            text="Check me"
+            :value="undefined"
```

Comment:
Please bind this value to a variable using two-way binding as in the Angular approach.

Reply:
fixed